src/org/authInfo.ts

```diff
@@ -98,7 +98,7 @@ export type OrgAuthorization = {
   instanceUrl?: string;
   accessToken?: string;
   error?: string;
-  isExpired: boolean;
+  isExpired: boolean | unknown;
```

Comment:
In my previous comment I meant that `isExpired` should be the literal string `'unknown'` similar to the `oauthMethod` property on line 93